docs/photoniq/event-delivery/clients/index.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: Event Delivery Client SDKs
+---
+
+This section demonstrates some of the tasks you can complete with JavaScript SDKs. Macrometa offers the following SDKs:
+
+- [WebSocket](/download/eds-clients/photoniq-eds-ws.min.js)
+- SSE (coming soon)
+
+## Pre-requisites
+
+- Download `photoniq-eds-ws.min.js` or `photoniq-eds-sse.min.js` from section above.
+- Attach the module to your html:
+```html
+<script src="path/to/photoniq-eds-ws.min.js"></script>
+<!-- or -->
+<script src="path/to/photoniq-eds-sse.min.js"></script>
+```
+
+## Basic Example
+
+Connect to Event Delivery Service, retrieve and subsribe to SQL:
+```js
+let config = {
+    host: "<YOUR-PHOTONIQ>.photoniq.macrometa.io",
+    customerId: "<YOUR-CUSTOMER-ID>",
+    apiKey: "<YOUR-API-KEY>",
+    fabric: "<YOUR-FABRIC>",
```

Comment:
While not really a doc comment...

Strange that you are exposing fabric here.
I would have thought that if you are using photoniq-auth, you would get the fabric that is tied to the apikey.

The same for customerId.  Does'nt the apikey identify the customerId?

Reply:
@joubert-macrometa - it is how now EDS works, `customerId` and `fabric` will be deleted when full functionality will be implemented in EDS. 
cc @manoj-macrometa

Reply:
Understood.